src/test/java/com/revature/chronicle/services/NotificationServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package com.revature.chronicle.services;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.revature.chronicle.daos.NotificationRepo;
+import com.revature.chronicle.models.Notification;
+import com.revature.chronicle.models.Ticket;
+
+@SpringBootTest
+class NotificationServiceTest {
+	
+	@Mock
+	private NotificationRepo notificationRepo;
+	
+	@InjectMocks
+	private NotificationService notificationService;
+	
+	@Before
+	public void setUp() {
+		MockitoAnnotations.openMocks(this);
+	}
+
+	@Test
+	void testGetNotified() {
```

Comment:
Is there some reason that you made a fresh ticket for each test? Were there side effects that you were trying to prevent here between tests?

Reply:
The reason for that is because of a couple of reasons: 
1: We have only been used to writing the tests this way, even though it is not the easiest way.
2: As I have stated before, I believe at least two different people in the previous iteration wrote these tests and that I believe was skilled with JUnit4 and the other was skilled in JUnit5. This caused a mixture of having some tests run and others not. So when we were first writing these tests we were still trying to solve this issue and follow the JUnit version that was in the pom.xml file. 

Since then we have been getting better and learning how to do it the easier way by studying the previous iterations code. So we have now just made a change to this test file so that we are using the easier method.